templates/plugin-distignore.mustache

```diff
@@ -21,7 +21,7 @@ phpunit.xml.dist
 multisite.xml
 multisite.xml.dist
 phpcs.xml
```

Comment:
You may also want to add `.phpcs.xml` here and possibly to the `.gitignore` file too ?

Reply:
@jrfnl thanks! I added .phpcs.xml but .gitignore is already in the file higher up.